zkevm-circuits/src/bytecode_circuit/bytecode_unroller.rs

```diff
@@ -92,6 +102,28 @@ impl<F: Field> Config<F> {
             push_rindex_inv,
         );
 
+        // Does the current row have bytecode field tag == Length?
+        let is_row_tag_length = |meta: &mut VirtualCells<F>| {
+            let row_tag = meta.query_advice(tag, Rotation::cur());
+            row_tag.expr() * (BytecodeFieldTag::Byte.expr() - row_tag.expr())
```

Comment:
Here you limit the `tag` to be either 0 or `BytecodeFieldTag::Byte`. But based on the `BytecodeFieldTag` in the `table.rs`, the tag value should be either 1 or 2, right?